vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# vars file for ansible-role-solr-prometheus-exporter
+prometheus_solr_exporter_config_location: "{{ prometheus_solr_exporter_installation_dir }}/contrib/prometheus-exporter/conf/solr-exporter-config.xml"
```

Comment:
Let's make this a default instead. I know the chances of it changing are slim, but I don't see any advantage to hard wiring it into a var that cant be changed by the consumer.

Reply:
Done